hack/deployer/config/plans.yml

```diff
@@ -40,7 +40,7 @@ plans:
   operation: create
   clusterName: ci
   provider: aks
-  kubernetesVersion: 1.16.7
+  kubernetesVersion: 1.18.2
```

Comment:
`1.18.2` is a preview version, any reason to not use `1.16.9` instead ?

![image](https://user-images.githubusercontent.com/976373/83638982-4e9a9b80-a5aa-11ea-812f-6729f35e92ef.png)

Reply:
Nop. I went too fast and didn't notice that it was still a preview version. Let's stick in 1.16.x.